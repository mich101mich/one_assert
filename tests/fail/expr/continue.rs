fn main() {
    for _ in 0..10 {
        one_assert::assert!(continue);
    }
}
