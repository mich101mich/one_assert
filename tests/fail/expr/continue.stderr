error: Expected a boolean expression, found a continue statement
 --> tests/fail/expr/continue.rs:3:29
  |
3 |         one_assert::assert!(continue);
  |                             ^^^^^^^^
